clients/include/testing_syrkx_batched.hpp

```diff
@@ -77,87 +70,148 @@ hipblasStatus_t testing_syrkx_batched(const Arguments& argus)
     int last = batch_count - 1;
     if(!dA || !dB || !dC || (!bA[last] && A_size) || (!bB[last] && B_size) || (!bC[last] && C_size))
     {
-        hipblasDestroy(handle);
         return HIPBLAS_STATUS_ALLOC_FAILED;
     }
 
+    double gpu_time_used = 0.0;
+    double hipblas_error = 0.0;
+
+    hipblasHandle_t handle;
+    hipblasCreate(&handle);
+
     // Initial Data on CPU
     srand(1);
     for(int b = 0; b < batch_count; b++)
     {
-        hA[b]  = host_vector<T>(A_size);
-        hB[b]  = host_vector<T>(B_size);
-        hC[b]  = host_vector<T>(C_size);
-        hC2[b] = host_vector<T>(C_size);
+        hA_array[b]      = host_vector<T>(A_size);
+        hB_array[b]      = host_vector<T>(B_size);
+        hC_array[b]      = host_vector<T>(C_size);
+        hC_array_copy[b] = host_vector<T>(C_size);
```

Comment:
Ask @daineAMD as maybe the host_batched_vector helpers have been ported over now?

Reply:
Yeah, the host_batched_vectors exist now. I don't mind going through with this PR now though, I've been making batched_vector changes along with other test changes to the entire test suite, I should be able to get to syrk in the next iteration.